plugins/reachability/src/main/java/org/qbicc/plugin/reachability/RTAInfo.java

```diff
@@ -360,10 +379,49 @@ synchronized void processInstantiatedClass(final LoadedTypeDefinition type, bool
         }
     }
 
+    synchronized void processServiceInterface(final LoadedTypeDefinition ltd, final String internalName) {
+        if (ltd.isInterface() && !isServiceSearched(ltd)) {
+            /* process service interface */
+            addReachableInterface(ltd);
```

Comment:
I don't think the interface would need to be marked reachable yet at this point.  If the results of the service loader are used, then the interface would be naturally reachable, otherwise we didn't need to do anything anyway.